src/sst/core/cfgoutput/pythonConfigOutput.cc

```diff
@@ -214,9 +215,26 @@ PythonConfigGraphOutput::generate(const Config* cfg, ConfigGraph* graph)
     // Generate the header and program options
     fprintf(outputFile, "# Automatically generated by SST\n");
     fprintf(outputFile, "import sst\n\n");
+    // We will dump all the program options so we can exactly recreate
+    // the run just by running the file.  The exceptions will be to
+    // information or configuration outputs.  We don't need to
+    // recreate output files when we run it again.  They're added in
+    // the order they're defined in the config.cc file.
+
     fprintf(outputFile, "# Define SST Program Options:\n");
+    fprintf(outputFile, "# (These reflect the settings from orininal run and are not necessary in all files)\n");
```

Comment:
"orininal"

Reply:
Have made the fix in the next set of changes.